webui.py

```diff
@@ -407,9 +410,11 @@ def process_images(outpath, func_init, func_sample, prompt, seed, sampler_name,
 
     os.makedirs(outpath, exist_ok=True)
 
-    sample_path = os.path.join(outpath, "samples")
+    if flat_outdir:

```

Comment:
is this meant to be the same?

Reply:
I didn't rigorously check the code since I wanted to confirm with you regarding the goals first ("bugs" during the specification phase always take the most effort to fix afterwards).

The implementation is supposed to be equivalent if `flat_outdir` is true.

Reply:
Ah, when I continued editing I understood what you meant.
This is a bug.
I misinterpreted what you were talking about because the conversation page only shows the lines _above_ your comment.